backend/src/main/java/com/crm/exception/validator/ValidSize.java

```diff
@@ -0,0 +1,48 @@
+package com.crm.exception.validator;
+
+import javax.validation.Constraint;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import javax.validation.Payload;
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import java.util.regex.Pattern;
+
+import static com.crm.exception.ErrorDict.*;
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@Target({FIELD})
+@Retention(RUNTIME)
+@Documented
+@Constraint(validatedBy = {ValidSize.ValidSizeValidator.class})
+public @interface ValidSize {
```

Comment:
Trochę duplikacja kodu. W sumie ma to samo co ValidPage. Może lepiej będzie jakoś to inaczej nazwać  i zastosować do size i page jeden interfejs?

Reply:
W sumie pomyślałem o tym, zmienię

Reply:
done